p2p/src/authenticated/actors/peer/actor.rs

```diff
@@ -102,14 +103,14 @@ impl<E: Spawner + Clock + ReasonablyRealtime + Rng + CryptoRng> Actor<E> {
                         content,
                     }
                 })),
-            };
+            }.encode_to_vec();
```

Comment:
You can do `.encode_to_vec().into()` and remove the `Bytes::from(..)` part below.